oaknational/Oak-Web-Application
# PUPIL-582: Add MathJax support to the pupil quiz

## Description

This ticket implements mathjax support in pupil lessons.

Richard can confirm the functionality.

@JBR90 I'd appreciate if you could do a code review as I think you have a bit of experience with mathjax and I'm also the only dev in my squad for a couple of weeks.  


## How to test

1. Go to https://deploy-preview-2348--oak-web-application.netlify.thenational.academy/pupils/programmes/maths-secondary-year-7/units/expressions-and-equations/lessons/simplifying-before-multiplying-with-multiple-expressions/exit-quiz
2. Goto question 4
3. You should be able to drag and drop mathjax items

_
1. Go to https://deploy-preview-2348--oak-web-application.netlify.thenational.academy/pupils/programmes/maths-secondary-year-7/units/expressions-and-equations/lessons/highest-common-factor-with-algebraic-terms/starter-quiz
2. The x & y values should look algebraic, right clicking them will show mathjax dialog.

_
1. Go to https://deploy-preview-2348--oak-web-application.netlify.thenational.academy/pupils/programmes/maths-secondary-year-7/units/expressions-and-equations/lessons/simplifying-before-multiplying-with-multiple-expressions/starter-quiz
2. Get question 5 wrong, the incorrect information at the bottom of the page should also show text in mathjax format.

## Checklist
